workarounds/stardewvalley.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Install SMAPI
+if [ -d "$extracted_scriptextender" ]; then
+	log_info "installing SMAPI in '$game_installation'"
+
+	# This command is ugly, but necessary 
+	# Needs to run through wineconsole in order to work with .NET's System.Console
+
+	"$utils/protontricks.sh" run-command "$game_appid" "wineconsole '${extracted_scriptextender}/SMAPI ${game_smapi_version} installer/internal/windows/SMAPI.Installer.exe' --install --no-prompt --game-path 'Z:\mnt\lingames\SteamLibrary\steamapps\common\Stardew Valley'" \
```

Comment:
The `--game-path` parameter seems to be hard-coded to a specific location in your system, which isn't a portable solution.
The "game_installation" variable set by [load_gameinfo.sh](https://github.com/rockerbacon/modorganizer2-linux-installer/blob/master/step/load_gameinfo.sh) will hold the install location. This should convert it to a Windows path:
```bash
echo "$game_installation" | sed -e 's/^/Z:/' -e 's@/@\\@g'
```

Reply:
Oops, the path was definitely a mistake on my part. Rushed from the yelling at the computer phase to the clean up for pull request phase. :stuck_out_tongue: